2020-11-04 11:54:53,940 ERROR [base_page.py 149] 元素--('xpath', "(//a[text()='更新提交'])[1]")--点击元素失败
2020-11-04 11:54:53,940 ERROR [base_page.py 150] Message: no such element: Unable to locate element: {"method":"xpath","selector":"(//a[text()='更新提交'])[1]"}
  (Session info: chrome=86.0.4240.183)
  (Driver info: chromedriver=71.0.3578.137 (86ee722808adfe9e3c92e6e8ea746ade08423c7e),platform=Mac OS X 10.14.6 x86_64)
Traceback (most recent call last):
  File "/Users/zhangcaiyan/Desktop/ketangpai/common/base_page.py", line 146, in click_element
    self.driver.find_element(*locator).click()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"(//a[text()='更新提交'])[1]"}
  (Session info: chrome=86.0.4240.183)
  (Driver info: chromedriver=71.0.3578.137 (86ee722808adfe9e3c92e6e8ea746ade08423c7e),platform=Mac OS X 10.14.6 x86_64)

2020-11-04 11:55:11,005 ERROR [base_page.py 149] 元素--('xpath', "(//a[text()='更新提交'])[1]")--点击元素失败
2020-11-04 11:55:11,006 ERROR [base_page.py 150] Message: no such element: Unable to locate element: {"method":"xpath","selector":"(//a[text()='更新提交'])[1]"}
  (Session info: chrome=86.0.4240.183)
  (Driver info: chromedriver=71.0.3578.137 (86ee722808adfe9e3c92e6e8ea746ade08423c7e),platform=Mac OS X 10.14.6 x86_64)
Traceback (most recent call last):
  File "/Users/zhangcaiyan/Desktop/ketangpai/common/base_page.py", line 146, in click_element
    self.driver.find_element(*locator).click()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"(//a[text()='更新提交'])[1]"}
  (Session info: chrome=86.0.4240.183)
  (Driver info: chromedriver=71.0.3578.137 (86ee722808adfe9e3c92e6e8ea746ade08423c7e),platform=Mac OS X 10.14.6 x86_64)

2020-11-04 11:55:28,102 ERROR [base_page.py 149] 元素--('xpath', "(//a[text()='更新提交'])[1]")--点击元素失败
2020-11-04 11:55:28,103 ERROR [base_page.py 150] Message: no such element: Unable to locate element: {"method":"xpath","selector":"(//a[text()='更新提交'])[1]"}
  (Session info: chrome=86.0.4240.183)
  (Driver info: chromedriver=71.0.3578.137 (86ee722808adfe9e3c92e6e8ea746ade08423c7e),platform=Mac OS X 10.14.6 x86_64)
Traceback (most recent call last):
  File "/Users/zhangcaiyan/Desktop/ketangpai/common/base_page.py", line 146, in click_element
    self.driver.find_element(*locator).click()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"(//a[text()='更新提交'])[1]"}
  (Session info: chrome=86.0.4240.183)
  (Driver info: chromedriver=71.0.3578.137 (86ee722808adfe9e3c92e6e8ea746ade08423c7e),platform=Mac OS X 10.14.6 x86_64)

2020-11-04 11:55:45,194 ERROR [base_page.py 149] 元素--('xpath', "(//a[text()='更新提交'])[1]")--点击元素失败
2020-11-04 11:55:45,194 ERROR [base_page.py 150] Message: no such element: Unable to locate element: {"method":"xpath","selector":"(//a[text()='更新提交'])[1]"}
  (Session info: chrome=86.0.4240.183)
  (Driver info: chromedriver=71.0.3578.137 (86ee722808adfe9e3c92e6e8ea746ade08423c7e),platform=Mac OS X 10.14.6 x86_64)
Traceback (most recent call last):
  File "/Users/zhangcaiyan/Desktop/ketangpai/common/base_page.py", line 146, in click_element
    self.driver.find_element(*locator).click()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"(//a[text()='更新提交'])[1]"}
  (Session info: chrome=86.0.4240.183)
  (Driver info: chromedriver=71.0.3578.137 (86ee722808adfe9e3c92e6e8ea746ade08423c7e),platform=Mac OS X 10.14.6 x86_64)

